clients/admin-ui/src/features/system/add-multiple-systems/AddMultipleSystems.tsx

```diff
@@ -218,7 +212,13 @@ export const AddMultipleSystems = ({ redirectRoute }: Props) => {
       pagination: {
         pageSize: PAGE_SIZES[0],
       },
+      columnSizing: {
+        select: 0,
```

Comment:
this allows these two columns to start off initially small (it doesn't start at 0, and admittedly I don't quite know how this works, but 0 seems to work ok)